### PR TITLE
Add example of deadmanssnitch setup.

### PR DESCRIPTION
**Pull request by gswallow**

### CODE REVIEW

File: Documentation/user-guides/configuring-prometheus-alertmanager.md [hunk 1]
```diff
@@ -142,9 +142,31 @@ receivers:
 
 ### Dead man's switch PagerDuty
 
-[PagerDuty][pagerduty] supports this mechanism through an integration called [Dead Man's Snitch][deadman-snitch]. Simply add a `PagerDuty` configuration to the default `deadmansswitch` receiver. Use the process described above to add this configuration.
+[PagerDuty][pagerduty] supports this mechanism through an integration called [Dead Man's Snitch][deadman-snitch].  Once you have signed up for Dead Man's Snitch and created a snitch, set up a webhook to talk to its unique URL:
 
-Configure Dead Man's Snitch to page the operator if the Dead man's switch alert is silent for 15 minutes. With the default Alertmanager configuration, the Dead man's switch alert is repeated every five minutes. If Dead Man's Snitch triggers after 15 minutes, it indicates that the notification has been unsuccessful at least twice.
+```
```
Comment (brancz) [inline Documentation/user-guides/configuring-prometheus-alertmanager.md]: This needs a yaml tag

File: examples/user-guides/configuring-prometheus-alertmanager/alertmanager-config-frontend-receiver.yaml [hunk 2]
```diff
@@ -19,6 +19,8 @@ route:
 receivers:
 - name: default
 - name: deadmansswitch
+  webhook_configs:
```
Comment (brancz) [inline examples/user-guides/configuring-prometheus-alertmanager/alertmanager-config-frontend-receiver.yaml]: This file is used for templating the section right before the DeadMansSnitch configuration section. Could you just create two versions of the files: one with, and one without the DeadMansSnitch configuration. Then we can embed both of them in the documentation.

Comment (gswallow) [reply examples/user-guides/configuring-prometheus-alertmanager/alertmanager-config-frontend-receiver.yaml]: What would you like to call it?

Comment (brancz) [reply examples/user-guides/configuring-prometheus-alertmanager/alertmanager-config-frontend-receiver.yaml]: let's just call it the same but with `-deadmanssnitch.yaml` suffix

File: Documentation/user-guides/configuring-prometheus-alertmanager.md [hunk 3]
```diff
@@ -142,9 +142,31 @@ receivers:
...
+receivers:
+- name: default
+  pagerduty_configs:
+  - service_key: "XXXXXX"
+- name: deadmansswitch
+  webhook_configs:
+    - url: "https://nosnch.in/XXXXXX
+```
+
+Configure a Dead Man's Snitch integration with PagerDuty, along with an excalation on PagerDuty to page the operator if the Dead man's switch alert is silent for 15 minutes. With the default Alertmanager configuration, the Dead man's switch alert is repeated every five minutes. If Dead Man's Snitch triggers after 15 minutes, it indicates that the notification has been unsuccessful at least twice.
```
Comment (mxinden) [inline Documentation/user-guides/configuring-prometheus-alertmanager.md]:
```suggestion
Configure a Dead Man's Snitch integration with PagerDuty, along with an escalation on PagerDuty to page the operator if the Dead man's switch alert is silent for 15 minutes. With the default Alertmanager configuration, the Dead man's switch alert is repeated every five minutes. If Dead Man's Snitch triggers after 15 minutes, it indicates that the notification has been unsuccessful at least twice.
```

**Comment by brancz** [comment]
Thanks a lot for adding this!

/ok-to-test
/lgtm
/approve

**Pull request merged**